creator-node/src/config.js

```diff
@@ -552,14 +550,23 @@ const config = convict({
     default: 5000
   },
   minimumSuccessfulSyncCountPercentage: {
-    doc: 'Minimum percentage of failed syncs to be considered unhealthy',
+    doc: 'Minimum percentage of failed syncs to be considered healthy',
     format: 'nat',
     env: 'minimumSuccessfulSyncCountPercentage',
     // TODO: Update to higher percentage when higher threshold of syncs are passing
     default: 0
+  },
+  minimumSecondaryUserSyncSuccessPercent: {
+    doc: 'Minimum percent of failed Syncs for a user on a secondary for the secondary to be considered healthy for that user',
+    format: 'nat',
+    env: 'minimumSecondaryUserSyncSuccessPercent',
+    default: 75
```

Comment:
this is likely what it's defaulted to for third party nodes, do we want it it at 50 or 75

Reply:
will set to 50 now for minimal impact rollout but the end goal for this should def be higher as this is at a per-user level